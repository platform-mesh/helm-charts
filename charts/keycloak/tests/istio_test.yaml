suite: istio
templates:
  - istio-virtualservice.yaml
release:
  namespace: openmfp-system
values:
  - ../test-values.yaml
tests:
  - it: matches the snapshot
    set:
      istio:
        enabled: true
        gateway:
          name: test
    asserts:
      - matchSnapshot: {}
  - it: matches the snapshot (with path prefix)
    set:
      domain.pathPrefix: /auth
      istio:
        enabled: true
        gateway:
          name: test
    asserts:
      - matchSnapshot: {}
