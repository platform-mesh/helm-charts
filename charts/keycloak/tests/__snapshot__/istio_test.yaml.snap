matches the snapshot:
  1: |
    apiVersion: networking.istio.io/v1beta1
    kind: VirtualService
    metadata:
      name: RELEASE-NAME-keycloak
    spec:
      gateways:
        - openmfp-system/test
      hosts:
        - keycloak.openmfp.org
      http:
        - match:
            - uri:
                prefix: /keycloak
          route:
            - destination:
                host: openmfp-keycloak.openmfp-system.svc.cluster.local
                port:
                  number: 80
matches the snapshot (with path prefix):
  1: |
    apiVersion: networking.istio.io/v1beta1
    kind: VirtualService
    metadata:
      name: RELEASE-NAME-keycloak
    spec:
      gateways:
        - openmfp-system/test
      hosts:
        - keycloak.openmfp.org
      http:
        - match:
            - uri:
                prefix: /auth
          route:
            - destination:
                host: openmfp-keycloak.openmfp-system.svc.cluster.local
                port:
                  number: 80
